### ISSUE TITLE
Visualizing Data using t-SNE

### ISSUE THREAD

**Issue by idigitopia** [idigitopia/idigitopia.github.io#4]
[paper](https://jmlr.org/papers/volume9/vandermaaten08a/vandermaaten08a.pdf), [demo1](http://projector.tensorflow.org/), [demo2](https://distill.pub/2016/misread-tsne/)

# Brief Thoughts: 
- Motivation: High dimensional data to low dimension (2 or 3) for visualization. 
- Can also be viewed as a clustering approach such that low-dimensional representations of very similar data points are kept close together. 
- preserve the local structure of the data, while revealing global structures. 
- use non-linear mapping of the underlying high-dimensional dataset instead of traditional linear approaches. 
- tunable parameters such as perplexity and learning rate. 
    - perplexity: conditional probability distribution induced by a gaussian kernel. 
    - intuitively it serves as  a way to balance attention between local and global aspects of the data. 
- it's good to visualize the data but not directly usable to better optimize the model or compare two representations. 
- cluster size and distance have relatively less meaning and is not grounded on the dataset.
